src/modules/navigator/mission_block.cpp

```diff
@@ -353,6 +353,32 @@ MissionBlock::set_loiter_item(struct mission_item_s *item, float min_clearance)
 void
 MissionBlock::set_takeoff_item(struct mission_item_s *item, float min_clearance, float min_pitch)
 {
```

Comment:
@jywilson This code adds a duplicate definition of MissionBlock::set_takeoff_item